.github/workflows/build.yml

```
@@ -9,177 +9,214 @@ on:
       - uses: actions/setup-python@v6
         with:
-          python-version: "3.11"
-          cache: "pip"
-      - name: Lint code
-        run: make ${{ env.FLAGS }} lint format
-  test:
-    name: Test ${{ matrix.os }}-py${{ matrix.python }}
-    runs-on: ${{ matrix.os }}
+          python-version: 3.11
+          cache: pip
+      - &install-emake
```

actionlint (1.7.12)

[error] anchor "install-emake" is defined but not used

(syntax-check)